### DISCUSSION TITLE
Second release

### DISCUSSION THREAD

**Discussion by EDsteve** [LIFsCode/ELOC-3.0#97] [General]
@LIFsCode @OOHehir 

Long time no see :) Hope everybody is doing well.

I am building a new batch of 25 ELOCs and will send it to the jungle in 3 weeks. It seems a good idea to make a new release for that reason. So the rangers then will also update the first batch with the new release. If that is fine with you?
I will add a few points for the new release in the issue section. But shouldn't be anything big.

Besides that I am training and testing models the last week and hope the new release will come with a few models we can test in the field.

After this release we will hopefully also have more LoRa experience (thx again for the [test code](https://github.com/LIFsCode/Lora_Test) @LIFsCode ) and if that looks promising, we can start with LoRa.
From a 15m tower we get 500-600m through thick jungle. Next test is from a 200m hill which should extend the coverage significantly.

If you see anything which makes sense to implment for the next release. Don't be shy :)

Cheeers
ED

**Comment by OOHehir** [comment]
@EDsteve @LIFsCode There are obviously some outstanding issues, from your side what do you think are the major ones?

Regarding the models you may wish to keep the sample rate the same as the I2S sample rate. I've been investigating the [ESP sound framework](https://docs.espressif.com/projects/esp-adf/en/v1.0/api-reference/index.html) which seems promising. However down-sampling the audio from 16K to something lower obviously require extra calculations & may outway any possible gains. I'll try putting together some tests & see what the results are like.

**Comment by OOHehir** [comment]
@EDsteve I forgot to ask is there any differences to the second board? An external crystal or the ability to assert the pins during flashing?

**Reply by EDsteve** [reply]
We still have 25 pcs from the last order. So the boards which gets deployed will be the same. [ELOC 3.3](https://github.com/LIFsCode/ELOC-3.0/files/13625126/Schematic_EL.OC_2023-11-17.pdf)
When i am in Germany (in one month) i will start testing with the external crystal.

**Comment by EDsteve** [comment]
@OOHehir @LIFsCode 
I will send new ELOCs to the jungle in Jambi (Sumatra) in two days. I am still working on the AI. But it will come with a model they can test.
I haven't seen any bigger issues with the last commit and therefore i think it makes sense to make a new release for that new update?

One small thing i would like to have in that release. Shouldn't have a big impact:
Only count detected events: #100

If you see anything else. Please let me know

**Comment by OOHehir** [comment]
@EDsteve I forgot about #96, I'll take a look at it this evening. Also take a look at #100.

**Comment by OOHehir** [comment]
@EDsteve Are you happy to tag the current main as a new release for the second batch of ELOC's? Just so we can mark the deployed code.

**Comment by EDsteve** [comment]
@OOHehir I have just published a new release (V1.1). That should have also tagged it automatically. I hope i did everything correct :)

**Comment by OOHehir** [comment]
@EDsteve Great, looks ok!
